src/bin/julialauncher.rs

```diff
@@ -430,11 +437,51 @@ fn run_app() -> Result<i32> {
         // process to handle things.
         ctrlc::set_handler(|| ()).with_context(|| "Failed to set the Ctrl-C handler.")?;
 
+        let mut job_attr: windows::Win32::Security::SECURITY_ATTRIBUTES =
+            windows::Win32::Security::SECURITY_ATTRIBUTES::default();
+        let mut job_info: windows::Win32::System::JobObjects::JOBOBJECT_EXTENDED_LIMIT_INFORMATION =
+            windows::Win32::System::JobObjects::JOBOBJECT_EXTENDED_LIMIT_INFORMATION::default();
+
+        job_attr.bInheritHandle = false.into();
+        job_info.BasicLimitInformation.LimitFlags =
+            windows::Win32::System::JobObjects::JOB_OBJECT_LIMIT_BREAKAWAY_OK
+                | windows::Win32::System::JobObjects::JOB_OBJECT_LIMIT_SILENT_BREAKAWAY_OK
+                | windows::Win32::System::JobObjects::JOB_OBJECT_LIMIT_DIE_ON_UNHANDLED_EXCEPTION
```

Comment:
This is a bug in libuv v1.x for compatibility reasons, but is fixed in libuv master
```suggestion
```